virttest/virsh.py

```diff
@@ -621,6 +624,93 @@ def kosher_args(remote_ip, uri):
         return True not in all_false
 
 
+class EventNotFoundError(Exception):
+    """
+    Error when certain event cannot be found.
+    """
+
+    def __init__(self, details=''):
+        self.details = details
+        Exception.__init__(self)
```

Comment:
I assume super is more straightforward when call parent __init__

Reply:
Let me check, actually I got this definition from LibvirtXMLError, but it seems not necessary now.

Reply:
It is unnecessary. I'll remove it since it's useless.